### ISSUE TITLE
Bump version of Bandit

### ISSUE THREAD

**Issue by oscar345** [sasa1977/site_encrypt#57]
The current version of Phoenix depends on a more recent version of Bandit, however, site_encrypt uses a older version of bandit. This makes it not possible to compile the application.

```
Resolving Hex dependencies...
Resolution completed in 0.112s
Because site_encrypt >= 0.5.0 depends on bandit ~> 0.6 and your app depends on bandit ~> 1.2, site_encrypt >= 0.5.0 is forbidden.
So, because your app depends on site_encrypt ~> 0.5, version solving failed.
** (Mix) Hex dependency resolution failed
```

**Comment by sasa1977** [comment]
I think the way to go is bumping the dep requirement to "~> 1.0" in site_encrypt too. PR is welcome :-)

**Comment by rkenzhebekov** [comment]
I guess we need v0.5.1? I was able to run the bandit 1.2.0 with the master branch. There is a deprecation warning `warning: UmitcareWeb.Endpoint.init/2 is deprecated,` but it is discussed in here: https://github.com/sasa1977/site_encrypt/issues/58

**Comment by sasa1977** [comment]
Published 0.5.1. The init deprecation requires more work and breaking changes, so I'll deal with that separately.
